examples/kitchen-sink/content/pages/home.md

```diff
@@ -28,5 +28,3 @@ blocks:
     _template: cta
 _template: blockPage
 ---
-
```

Comment:
@jamespohalloran should all these changes to the example projects be in this PR?

Reply:
that's weird.. when I run `yarn pretty-quick staged` on main there are no changes and same thing on your branch

Reply:
you can run this and commit to revert the changes:
```
git checkout origin/main experimental-examples
git checkout origin/main examples
```